simplq/src/utils/scrollingOperations.js

```diff
@@ -1,3 +1,5 @@
+import './scrollingOperation.css';
+
```

Comment:
This is a tiny detail, but is there a way we could stick to the convention of using scss files and still have this work? This single css file just looks a little out of place. Not a big deal, if it's not a simple change, just leaving a comment below about why this was done this way is good enough IMO.

Reply:
I have also solved it in my latest issue